chemprop/args.py

```diff
@@ -490,6 +577,24 @@ def num_tasks(self) -> int:
         """The number of tasks being trained on."""
         return len(self.task_names) if self.task_names is not None else 0
 
+    @property
+    def quantiles(self) -> List[float]:
+        """A list of quantiles to be being trained on."""
+        return self._quantiles
+
+    @quantiles.setter
+    def quantiles(self, quantiles: List[float]) -> None:
+        self._quantiles = quantiles
+
+    @property
+    def quantiles_tensor(self) -> List[float]:
+        """A list of quantiles to be being trained on as torch tensor."""
+        return self._quantiles_tensor
+
+    @quantiles_tensor.setter
+    def quantiles_tensor(self, quantiles_tensor: List[float]) -> None:
+        self._quantiles_tensor = quantiles_tensor
+
```

Comment:
typings here indicate lists of floats, but the instance where they are set in L65 of crossvalidate.py is actually assigning a tensor.

These args get saved and loaded in a lot of contexts. I don't have a specific example, but I'd worry that having a complex type like tensor as a saved arg will cause issues in saving the args.json or similar.

Reply:
Are these setter/getter structures really necessary?

Reply:
Hey Charles, this is a really good point, I'll change it to this. I think if we get rid of quantiles_tensor, we can just keep quantiles as a setter/getter?